src/Psalm/ErrorBaseline.php

```diff
@@ -273,6 +274,10 @@ function (string $extension) : string {
                 foreach ($existingIssueType['s'] as $selection) {
                     $codeNode = $baselineDoc->createElement('code');
 
+                    /** @todo in major version release (e.g. 5.0.0) replace $selection with trim($selection)
```

Comment:
```suggestion
                    /** @todo in major version release (e.g. Psalm 5) replace $selection with trim($selection)
```
We've been marking those things with 'Psalm 5' so that when the time comes they could all be uncovered with a single `grep 'Psalm 5'` run.